src/common/utils.js

```diff
@@ -201,8 +202,19 @@ const getMerkleProofForAccount = (account, snapshot) => {
 	return proof
 }
 
+const convertBlocksToDays = (blocks) => {
+	return blocks / (EST_BLOCKS_PER_HOURS * 24)
+}
+
+const getTokenPrice = (tokenId) => {
+	return fetch(`${defaults.api.coingeckoUrl}?vs_currencies=usd&ids=${tokenId}`)
```

Comment:
What kind of tokens will be priced using this API?

Reply:
Principal & payout tokens. The prices are needed for calculating ROI, treasury balance..